pandas/core/categorical.py

```diff
@@ -1995,14 +1995,18 @@ def is_dtype_equal(self, other):
 
         Parameters
         ----------
-        other : Categorical
+        other : Categorical, Series
 
         Returns
         -------
         are_equal : boolean
         """
-
         try:
+            from pandas.core.series import Series
+
+            if isinstance(other, Series):
```

Comment:
Doing `Categorical(thing, ordered=False)` and having it come out ordered would be very odd. If we do this, we would need to change the default to `ordered=None`.